Makefile

```diff
@@ -112,7 +112,7 @@ all: crane-manager metric-adapter
 
 .PHONY: crane-manager
```

Comment:
if decide to change crane-manager to craned, please replace them all for whole project including dirs and files